etc/alignak.cfg

```diff
@@ -135,6 +128,25 @@ max_plugins_output_length=65536
 #additional_freshness_latency=15
 
 
+# Checks scheduler configuration
+# ---
+# Interval length and re-scheduling configuration
+# Do not change those values unless you are really sure to master what you are doing...
+# todo: confirm the real interest of those configuration parameters!
+#interval_length=60
+#auto_reschedule_checks=1
+#auto_rescheduling_interval=1
+#auto_rescheduling_window=180
+
+# Number of interval to spread the first checks for hosts and services
+# Default is 30
+#max_service_check_spread=30
+max_service_check_spread=5
```

Comment:
This PR did not modified anything to the default configuration parameters of Alignak. This configuration exists in that form since a very long time 😉 

As such, I do not agree changing this with this PR. If you find it not logical, open an issue to update the default configuration shipped with Alignak.